elements/pl-matching/pl-matching.mustache

```diff
@@ -0,0 +1,110 @@
+<script>
+    $(function(){
+        $('#pl-matching-{{uuid}} [data-toggle="popover"]').popover({
+            sanitize: false,
+            container: 'body',
+            template: '<div class="popover pl-dropdown-popover" role="tooltip"><div class="arrow"></div><h3 class="popover-header"></h3><div class="popover-body"></div></div>',
+        });
+    });
+</script>
+<style>
+.matching-options ol > li:before {
+    content: counter(item, {{counter_type}}) ".";
+}
+</style>
+{{#question}}
+<div id="pl-matching-{{uuid}}" style="display: flex;">
+<div class="matching-answers">
+{{#answers}}
+    <div>
+        <div class="matching-answer-select">
+            <select name="{{name}}" class="form-control">
+                {{#options}}
+                    <option {{selected}} value="{{value}}">{{value}}</option>
+                {{/options}}
+            </select>
+        </div>
+        <div>
+            {{{html}}}
+        </div>
+        {{#display_score_badge}}
+        <div>
+            {{#correct}}
+                <span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i></span>
+            {{/correct}}
+            {{^correct}}
+                <span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i></span>
+            {{/correct}}
+        </div>
+        {{/display_score_badge}}
+    </div>
+{{/answers}}
+</div>
+<div class="matching-options">
+<ol>
+{{#options}}
+    <li class="matching-option">{{{html}}}</li>
+{{/options}}
+</ol>
+</div>
+</div>
+{{#correct}}&nbsp;<span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>{{/correct}}
+{{#partial}}&nbsp;<span class="badge badge-warning"><i class="far fa-circle" aria-hidden="true"></i> {{partial}}%</span>{{/partial}}
+{{#incorrect}}&nbsp;<span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i> 0%</span>{{/incorrect}}
+{{/question}}
+
+{{#submission}}
+<div id="pl-matching-{{uuid}}" style="display: flex;">
+<div class="matching-answers">
+{{#answers}}
+    <div>
+        <div class="matching-answer-select">
+            <select disabled class="form-control">
+                {{#options}}
+                    <option {{selected}} value="{{value}}">{{value}}</option>
+                {{/options}}
+            </select>
```

Comment:
Could this be just [formatted] text instead of a disabled select? This would make it more consistent with other elements.

Reply:
How would you prefer it to be formatted? Maybe bold and in parentheses, like **(a)** ?

Reply:
Whatever you think looks better. Bold with either parenthesis or followed by a period and space should be fine too.